python/mcap/mcap/mcap0/reader.py

```diff
@@ -142,6 +146,7 @@ def iter_messages(
         topics: Optional[Iterable[str]] = None,
         start_time: Optional[int] = None,
         end_time: Optional[int] = None,
+        log_time_order: bool = False,
```

Comment:
I've changed the default to True, but for posterity: even for seekable streams you need to load all messages into memory if there is no summary section on the MCAP. This might constitute a performance problem, so I wanted the default to cause the least surprises.